.travis.yml

```diff
@@ -4,10 +4,13 @@ install:
   - pip install -r requirements.txt
   - pip install pytest==5.2.0
   - python -m spacy download en_core_web_sm
+  - wget -O src/relation_extraction_model/transformer_dir/tf_model.h5 https://storage.googleapis.com/coronaviruspublicdata/re_final_best/new/tf_model.h5
```

Comment:
this should be taken care of programmatically with python. Use ` from urllib.request import urlretrieve` for similar functionality inside your code